h2o-py/h2o/model/model_base.py

```diff
@@ -996,12 +996,12 @@ def download_mojo(self, path=".", get_genmodel_jar=False, genmodel_name=""):
                 h2o.api("GET /3/h2o-genmodel.jar", save_to=os.path.join(path, genmodel_name))
         return h2o.api("GET /3/Models/%s/mojo" % self.model_id, save_to=path)
 
-    def save_mojo(self, path="", force=False):
+    def save_mojo(self, path="", filename=None, force=False):
```

Comment:
save_mojo and download_mojo should have similar capabilities and semantics - is it still the case?

Reply:
Solved on call